db/migrate/20221020114402_add_user_ref_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, null: false, foreign_key: { to_table: :users }
+    add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
+  end
+end
```

Comment:
- [ ] Kindly check this error that you are getting from this migration
![image](https://user-images.githubusercontent.com/62705947/196987949-cfd2de50-493e-4cd4-bdd5-27737eaef08e.png)

Reply:
@GzimAsani  Thank you for reviewing my project, I will make the necessary corrections as suggested
I deleted the duplicated reference in likes and comments